app/views/embassies/index.html.erb

```diff
@@ -23,21 +23,21 @@
               } %>
             </div>
             <div class="govuk-grid-column-two-thirds">
-              <% if embassy.has_consular_service_in_location? && embassy.offices.empty? %>
-                <p class="govuk-body"><%= embassy.text %></p>
-              <% else %>
+              <% if embassy.can_assist_british_nationals? %>
                 <ul class="govuk-list govuk-list--spaced govuk-!-margin-top-0">
-                  <% if !embassy.has_consular_service_in_location? || embassy.has_remote_service? %>
-                  <li>
-                    <p class="govuk-body">
-                      <%= embassy.text %>
-                    </p>
-                    <%= embassy.embassy_path %>
-                  </li>
+                  <% if embassy.can_assist_in_location? %>
+                    <%= render partial:"organisation", collection: embassy.organisations_with_embassy_offices %>
```

Comment:
Very minor: I know you didn't add this, but it would be nice to take the opportunity to add a space between `partial:` and "organisation" here!

Reply:
Fixed in 7692345ab2